charms/kubernetes_snaps.py

```diff
@@ -211,10 +199,10 @@ def configure_apiserver(
 def configure_controller_manager(
     cluster_cidr,
     cluster_name,
+    external_cloud_provider: ExternalCloud,
     extra_args_config,
     kubeconfig,
     service_cidr,
-    external_cloud_provider: ExternalCloud,
```

Comment:
i'm ok if the arg order needs to change -- but we'd have to adjust the caller in kubernetes-control-plane